robot/rospackages/src/mcu_control/scripts/PdsNode.py

```diff
@@ -94,40 +94,40 @@ def init_serial():
     elif uart:
         port = 'ttySAC0'
         rospy.loginfo('Attempting to connect to /dev/' + port)
+
         try:
             ser = serial.Serial('/dev/' + port, baudrate)
         except:
             rospy.logerr('No UART device recognized, terminating PDS node')
             sys.exit(0)
 
         rospy.loginfo("clearing buffer...")
+
         while ser.in_waiting:
             ser.readline()
 
         rospy.loginfo("identifying MCU by sending 'who' every %d ms", timeout * 1000)
+
         for i in range(1, 6):
             rospy.loginfo('attempt #%d...', i)
             startListening = time.time()
             ser.write(str.encode('who\n'))
+
             while time.time() - startListening < timeout:
                 if ser.in_waiting:
                     dat = ''
                     data = None
                     try:
-                        #dat = ser.readline().decode()
                         data = ser.readline().decode()
-                        #data = stripFeedback(dat)
                     except Exception as e:
                         print("type error: " + str(e))
                         rospy.logwarn('trouble reading from serial port')
+
                     if data is not None:
                         if mcuName in data:
                             rospy.loginfo(mcuName + " MCU identified!")
                             rospy.loginfo('timeout: %f ms', (time.time() - startListening) * 1000)
-                            rospy.loginfo(
-                                'took %f ms to find the ' + mcuName + ' MCU',
-                                (time.time() - startConnecting) * 1000
-                            )
+                            rospy.loginfo('took %f ms to find the ' + mcuName + ' MCU', (time.time() - startConnecting) * 1000)
                             return
                     else:
                         rospy.loginfo('got raw message: ' + dat)
```

Comment:
fyi if you make more commits on this pr dont touch this part im changing everything in #282